chapters/lang/items.tex

```diff
@@ -14,7 +14,7 @@ \section{Functions}
   \gd{funparams} \eq \term{\gp{id}} \ \gmany{ \ \term{,} \ \term{\gp{id}} \ } \ \gtry{\term{,}}
 \end{bnf}
 
-A \emph{function}\index{item!function} is a named block, along with optional set of parameters. Functions are declared with the keyword \lstinline{fun}. Function returns the result of evaluating of contained block. Functions may declare a set of input variables as parameters, through which the caller passes arguments into the function. Input variables behave the same as normal variables defined within function block.
+A \emph{function}\index{item!function} is a named block, along with optional set of parameters. Functions are declared with the keyword \lstinline{fun}. If keyword \emph{return} is not used, function returns the result of evaluating last expression from contained block. Functions may declare a set of input variables as parameters, through which the caller passes arguments into the function. Input variables behave the same as normal variables defined within function block.
```

Comment:
This sentence is false, keyword return may be used, and function will still return last expression:

```
fun f() {
  if False { return 5; }
  3
}
```

Reply:
But there is nothing about behavior when using return statement. It is said that when NOT using that it doing sth.